tests/jerry/func-decl.js

```diff
@@ -38,3 +38,18 @@ check_syntax_error ("'use strict'; function arguments () {}");
 check_syntax_error ("'use strict'; var l = function arguments () {}");
 
 check_syntax_error ("function f__strict_mode_duplicate_parameters (p, p) { 'use strict'; }");
+
+function test_strict_mode_propagation_in_func_expr_and_getters_setters () {
```

Comment:
@egavrin, yes I was thinking about something like that (aside from the fact that it may not throw syntax error but something else)